kdestandardize.ado

```diff
@@ -170,6 +170,15 @@ prog def kdestandardize
 		la def prtargettype 1 "Number Proficient/Distinguished"				 ///   
 						2 "Percent Proficient/Distinguished", modify
 						
+		// Define Equity Measures
```

Comment:
We'll need to make these 7 values value labels for 7 new variables and denormalize the values related to these to make the file more useful/helpful.